src/ui/commandpalette.ts

```diff
@@ -241,31 +241,10 @@ namespace CommandItem {
 export
 class CommandPalette extends Widget {
   /**
-   * Create the DOM node for a command palette.
-   */
-  static createNode(): HTMLElement {
-    let node = document.createElement('div');
-    let search = document.createElement('div');
-    let wrapper = document.createElement('div');
-    let input = document.createElement('input');
-    let content = document.createElement('ul');
-    search.className = SEARCH_CLASS;
-    wrapper.className = WRAPPER_CLASS;
-    input.className = INPUT_CLASS;
-    content.className = CONTENT_CLASS;
-    input.spellcheck = false;
-    wrapper.appendChild(input);
-    search.appendChild(wrapper);
-    node.appendChild(search);
-    node.appendChild(content);
-    return node;
-  }
-
-  /**
    * Construct a new command palette.
    */
   constructor(options: CommandPalette.IOptions = {}) {
-    super();
+    super({ node: Private.createNode() });
```

Comment:
Exactly. The widget relies on the structure of the node. External code can customize the dynamic content, but not the static content.